src/components/lastUpdated/tests/LastUpdated.spec.tsx

```diff
@@ -0,0 +1,112 @@
+import { shallow, mount, ReactWrapper, ShallowWrapper } from 'enzyme';
+import { LastUpdated, ILastUpdatedProps, defaultLastUpdateLabel } from '../LastUpdated';
+import { TestUtils } from '../../../utils/TestUtils';
+import * as s from 'underscore.string';
+import * as moment from 'moment';
+
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+
+describe('LastUpdated', () => {
```

Comment:
I don't think you need to have 2 describes.

I would only do `describe('<LastUpdated>', () => {...});`.